stubs/Nette/ComponentModel/Component.php

```diff
@@ -4,6 +4,10 @@
 
 namespace Nette\ComponentModel;
 
+if (class_exists('Nette\ComponentModel\Component')) {
+    return;
+}
```

Comment:
:+1:

Reply:
It should be 👍 but it doesn't work! So it's 👎 then 😛